.github/workflows/test_api_lint.yml

```diff
@@ -0,0 +1,28 @@
+name: Lint
+
+on:
+  pull_request:
+
+jobs:
+  lint:
+    name: Pr checks
+    runs-on: ubuntu-latest
```

Comment:
don't use latest. Use specific version

Reply:
That's a good point, i'll change that. We are using the latest though in the user repo actions

Reply:
Not in `node`. But we should change is everywhere

Reply:
ah ok